apps/web/frontend/components/organizationMemberList/organizationMemberList.tsx

```diff
@@ -58,23 +58,10 @@ export const OrganizationMemberList = (props: OrganizationMemberListProps) => {
         {organization.members.map((user) => (
           <tr key={user.id}>
             <td className="flex w-full items-center gap-2">
-              {user.image ? (
-                <div className="avatar">
-                  <Image
-                    className="rounded-box"
-                    width={64}
-                    height={64}
-                    src={user.image}
-                    alt={user.name ?? 'image of the user'}
-                  />
-                </div>
-              ) : (
-                <div className="avatar placeholder">
-                  <div className="size-16 rounded-box bg-neutral text-neutral-content" />
-                </div>
-              )}
-              {user.name}
-              <OrganizationRoleLabel role={user.organizationRole} />
+              <div className="min-w-52">
+                <UserLabel name={user.name ?? user.id} image={user.image ?? undefined} imageSize={28} />{' '}
+              </div>
```

Comment:
```suggestion
                <UserLabel name={user.name ?? user.id} image={user.image ?? undefined} imageSize={28} />{' '}
```

I believe it makes more sense for a badge to "stick" to a project member rather than being aligned vertically. However, if you'd like to stick to current implementation, it might be better to add a separate column instead of using an arbitrary width.

Reply:
We tried to separate them in a single column, but the space between them was too big.  We decided to keep the min-w-52 after we tested it by adding three names.